packages/elements/src/components/API/APIWithStackedLayout.tsx

```diff
@@ -61,7 +63,7 @@ export const APIWithStackedLayout: React.FC<StackedLayoutProps> = ({
             nodeTitle={serviceNode.name}
             nodeType={NodeType.HttpService}
             location={location}
-            layoutOptions={{ showPoweredByLink: true, hideExport }}
+            layoutOptions={{ showPoweredByLink, hideExport }}
```

Comment:
is this something we actually want to conditionally control?

Reply:
As discussed since this is internal component limited to stacked layout we might have that conditional control